### DISCUSSION TITLE
VLC Media Player

### DISCUSSION THREAD

**Discussion by JuliaAgostino** [mayrinck/FOSSonMicrosoftStore#8] [Requests] [answered]
VLC Media Player website: https://www.videolan.org/vlc/

VLC Media Player privacy policy: https://www.videolan.org/privacy.html

Logo: https://upload.wikimedia.org/wikipedia/commons/e/e6/VLC_Icon.svg

Description: "VLC media player (previously the VideoLAN Client and commonly known as simply VLC) is a free and open-source, portable, cross-platform media player software and streaming media server developed by the VideoLAN project. VLC is available for desktop operating systems and mobile platforms, such as Android, iOS, iPadOS, Windows 10 Mobile, and Windows Phone. VLC is also available on digital distribution platforms such as Apple's App Store, Google Play, and Microsoft Store.

VLC supports many audio and video compression methods and file formats, including DVD-Video, video CD and streaming protocols. It is able to stream media over computer networks and can transcode multimedia files.

The default distribution of VLC includes many free decoding and encoding libraries, avoiding the need for finding/calibrating proprietary plugins. The libavcodec library from the FFmpeg project provides many of VLC's codecs, but the player mainly uses its own muxers and demuxers. It also has its own protocol implementations. It also gained distinction as the first player to support playback of encrypted DVDs on Linux and macOS by using the libdvdcss DVD decryption library; however, this library is legally controversial and is not included in many software repositories of Linux distributions as a result."

**Comment by mayrinck** [comment, marked as answer]
I don't think we need/should post VLC over the store, they have [their own version](https://www.microsoft.com/en-us/p/vlc/9nblggh4vvnh?activetab=pivot:overviewtab) there... I know is not as powerful as the classic VLC but it's official.

**Reply by Justin-Vega** [reply]
Their own version is UWP and doesn't support DVDs or Blu-rays.

**Reply by mayrinck** [reply]
> 
> 
> Their own version is UWP and doesn't support DVDs or Blu-rays.

Exactly, but it's official. We will support software that is not already there officially.
